integration/spark-datasource/src/main/scala/org/apache/spark/sql/carbondata/execution/datasources/CarbonSparkDataSourceUtil.scala

```diff
@@ -269,4 +273,94 @@ object CarbonSparkDataSourceUtil {
     }
     model
   }
+
+  def validateTableOptions(options: Map[String, String], schema: Schema): Unit = {
```

Comment:
I don't expect any validations here. We cannot validate all the properties during load. We can do validation only for the properties which it supports. And also those validations should be inside sdk interfaces not here.